Jenkinsfile

```diff
@@ -171,6 +181,8 @@ pipeline {
                                 return
                             }
 
+                            openshift.tag("--source=docker", "${imageName}", "${openshift.project()}/${params.IMAGE_STREAM_NAME}:latest")
```

Comment:
I think we want to reference `imagestream_latest_tag` instead of hardcoding `latest`